episodes/excthreesteps.Rmd

```diff
@@ -25,6 +25,20 @@ Open terminal, activate `mirVE` environment and run python
 conda activate mirVE
 python
 ```
+:::::::::::::::::::::::::::::::::::::: spoiler
+
+## How to avoid `ModuleNotFoundError: No module named VARIABLE`?
+
+You need to export relative imports, using the following notation for either linux or windows (might be similar to mac)
+
+```bash
+#WINDOWS
+set PYTHONPATH=%PYTHONPATH%;C:\path\to\your\project\
+#LINUX
+export PYTHONPATH="${PYTHONPATH}:/path/to/your/project/"
+```
+
+::::::::::::::::::::::::::::::::::::::

```

Comment:
I'm very cautious about using PYTHONPATH in conjunction with conda, as that PYTHONPATH will stay the same regardless of what environment you are using, so any modules in PYTHONPATH could end up clashing with different versions and modules that you have loaded in various environments. I'm not sure whether we should ahve this in the text